include/aspect/geometry_model/box.h

```diff
@@ -71,6 +71,13 @@ namespace aspect
         virtual
         Point<dim> get_extents () const;
 
+        /**
+         * Return an integer array that denotes the number of repetitions of
+         * the box's coarse mesh.
+         */
+        std::array<unsigned int,dim>
+        get_repetitions () const;
```

Comment:
If you write this as:
```suggestion
        const std::vector<unsigned int> &
        get_repetitions () const;
```
You can make the function much easier (see my comment below). (also update the comment above when you do this).

Reply:
This comment still applies.